dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTableTest.java

```diff
@@ -0,0 +1,32 @@
+package com.dotmarketing.startup.runonce;
+
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task201016UpdatePrimaryKeyLengthIdentifierTableTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)